packages/bpk-component-image/src/BpkImage.js

```diff
@@ -42,39 +42,62 @@ type BpkImageProps = {
 
 type ImageProps = {
   altText: string,
-  hidden: boolean,
+  hidden: ?boolean,
   onImageLoad: ?() => mixed,
 };
 
-const Image = (props: ImageProps): Node => {
-  const { hidden, altText, onImageLoad, ...rest } = props;
+class Image extends Component<ImageProps> {
+  img: ?HTMLImageElement;
 
-  const imgClassNames = [getClassName('bpk-image__img')];
+  static propTypes = {
+    altText: PropTypes.string.isRequired,
+    hidden: PropTypes.bool,
+    onImageLoad: PropTypes.func.isRequired,
```

Comment:
`onImageLoad` is required in props but optional in flow types .. which is it?

Reply:
I think it was like this before, I only moved props and flow types around. Made it required now.